### ISSUE TITLE
Does [MASK] not work in the chatGLM series?

### ISSUE THREAD

**Issue by Michelleable** [THUDM/ChatGLM2-6B#188]
### Is there an existing issue for this?

- [ ] I have searched the existing issues

### Current Behavior

I wanna use chatGLM like GLM with [MASK] to do blank filling, but it seems to be unsuccessful. 

### Steps To Reproduce

just like " The weather is really [MASK]. It's so cold."

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by duzx16** [comment]
The chat model is fine-tuned with causal mask without blank filling.

**Comment by Michelleable** [comment]
> The chat model is fine-tuned with causal mask without blank filling.

Thanks for your answering so much. But I have some questions still.
"finetuned with causal mask", but was it still pretrained with "mask and gmask"?
 If so, I'm confused what the position embedding is with [MASK] in 1D position embedding

In GLM-130B paper,"mask sub-matrices [MASK] generation also indicate the token order"  , but I'm confused.
Like this?
The movie is a little boring, so I don't like it.
The movie is [MASK], so I don't like it.
0 1 2 [3] 4 5 6 ... [sop] 3 3 3 3 ?
or 0 1 2 [3] 4 5 6 ... [sop] 7 8 9 ?

Looking forward to your reply，thanks!
